themes/nord.json

```diff
@@ -14,10 +14,8 @@
     "button.background": "#434c5e",
     "button.foreground": "#d8dee9",
     "button.hoverBackground": "#4c566a",
-    "diffEditor.insertedTextBackground": "#a3be8c4d",
-    "diffEditor.insertedTextBorder": "#a3be8c4d",
+    "diffEditor.insertedTextBackground": "#a3eeaa2d",
```

Comment:
Let's get this done and change it to `#81a1c133` so we can leave these `nord14` opacity-variations behind.